modules/custom-status/custom-status.php

```diff
@@ -466,7 +478,7 @@ public function add_custom_status( $term, $args = [] ) {
 			$args['position'] = $default_position;
 		}
 
-		$encoded_description = $this->get_encoded_description( $args );
+		$encoded_description = TaxonomyUtilities::get_encoded_description( $args );
```

Comment:
`TaxonomyUtilities` only currently handles `get_encoded_description()` and `get_unencoded_description()`, but this allowed moving some repeated code out of the `CustomStatus` and `EditorialMetadata` classes.